go-staticcheck.toml

```diff
@@ -81,6 +81,16 @@ rewrite='!strings.Contains(:[1], :[2])'
 match='strings.Index(:[1], :[2]) < 0'
 rewrite='!strings.Contains(:[1], :[2])'
 
+############################################
+# https://staticcheck.io/docs/checks#S1002 #
+############################################
+[S1002_01]
+match='if :[1] == true'
```

Comment:
For this and other patterns, it's OK to do this, but in some older versions of comby this could be expensive, where `:[1]` tries to match the entire rest of file. In the current version, it will only match up to the end of line. To get the previous behavior that matches across newlines, it's possible to add `-match-newline-at-toplevel` on the CLI. At some point I will add a rule to give more control over this behavior so it can be adjusted in this file. Just pointing out this behavior.

Reply:
thank you for the highlight